src/postprocessors/TallyRelativeError.C

```diff
@@ -86,20 +86,24 @@ TallyRelativeError::getValue() const
 {
   const auto & tally = _openmc_problem->getLocalTally();
 
-  Real extreme_value;
+  Real post_processor_value;
 
   switch (_type)
   {
     case operation::max:
-      extreme_value = std::numeric_limits<Real>::min();
+      post_processor_value = std::numeric_limits<Real>::min();
       break;
     case operation::min:
-      extreme_value = std::numeric_limits<Real>::max();
+      post_processor_value = std::numeric_limits<Real>::max();
+      break;
+    case operation::average:
+      post_processor_value = 0.0;
       break;
     default:
       mooseError("Unhandled OperationEnum!");
   }
 
+  unsigned int num_values = 0u;
```

Comment:
I don't think you need this `0u`, but `0` would work fine. Is there a specific scenario you were thinking of by using `0u`, or just habit?

Reply:
It's a habit of mine from other projects with styles that abuse `auto`. I can change it to `0` and try to break the habit for future PRs if that's preferred.

Reply:
I think convention in MOOSE is to use type-agnostic syntax as much as possible, so I'd prefer to remove it. I think you can also safely remove the `static_cast<Real>` line later as well, I think the type conversion should happen fine without it (but lmk if that's not the case).

Reply:
Pushed style changes. Tests are still passing on my end so it looks like the `static_cast` was unnecessary.